frontend/src/app/matching/matching.component.ts

```diff
@@ -45,7 +45,7 @@ export class MatchingComponent {
   }
 
   cancelMatch() {
-    let userid = this.storageService.getUser()["id"]
+    let userid = this.storageService.getUser()["username"]
```

Comment:
Change should be made elsewhere if relevant to functionality of deployed software

Reply:
Yeah this was a change I made to fix the id thing I mentioned, its probably better to be included in the bug fixes with @cjyothika's current PR. I'll remove it from here and let her know.